Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'berkshelf', '~> 5.1', '>= 5.1.0'
 
 group :unit do
   gem 'foodcritic',       '~> 8.1', '>= 8.1.0'
-  gem 'rubocop',          '~> 0.45', '>= 0.45.0'
+  gem 'cookstyle'
```

Comment:
Cookstyle lets you avoid all rules that make zero sense in a cookbook. It simply avoids users having to start with a 30 lines .rubocop.yml when they create a new cookbook. As time goes on we expect we'll add the logic of foodcritic into cookstyle as rubocop plugins so it just becomes the linting for both Chef DSL and Ruby.